Utils/AppConstants.swift

```diff
@@ -10,13 +10,28 @@ public enum AppBuildChannel {
     case Release
 }
 
+/**
+ * MOZ_FLAGS
+ *
+ * MOZ_LOGIN_MANAGER - Toggles the Logins manager from appearing in the Settings.
+ *
+ */
+
 public struct AppConstants {
 #if MOZ_CHANNEL_AURORA
     public static let BuildChannel = AppBuildChannel.Aurora
+
+
+    public static let MOZ_LOGIN_MANAGER = false
```

Comment:
This is fine, but in the future you might consider inverting these, which is what we do on Android:

```
public static let MOZ_LOGIN_MANAGER =
#if MOZ_CHANNEL_AURORA
false
#else
…
#endif
```

When these flags proliferate you increase the chances of you writing code that doesn't compile with some sets of flags due to omitted or misnamed vars in one of these branches.